make_deb

```diff
@@ -72,15 +72,20 @@ then
   exit 1
 fi
 
-version=`cat package.json | grep version | head -1 | awk -F: '{ print $2 }' | sed 's/[" ,]//g'`
+version=$(cat package.json | grep version | head -1 | awk -F: '{ print $2 }' | sed 's/[" ,]//g')
 BUILD_ROOT="${GIT_ROOT}/packages/build"
 
+# Get git info
+git_tag_info=$(git describe --tags) && commit_count=$(echo ${git_tag_info} | cut -d- -f2)
+branch_name=$(git rev-parse --abbrev-ref HEAD)
+# If tagged version is checked out then branch name is HEAD; set it to tag version value
+if [[ $branch_name == "HEAD" ]]; then branch_name=$(git describe --tags); fi
 # Final package name
-package_name="fledge-display-${version}"
+if [[ ${branch_name} != "main" ]] && [[ ! ${branch_name} =~ ^[0-9]+\.[0-9]+\.[0-9]+RC ]] && [[ ! ${branch_name} =~ ^v[0-9]+\.[0-9]+\.[0-9]+ ]]; then package_name="fledge-display_${version}-${commit_count}"; version=${git_tag_info:1}; else package_name="fledge-display${version}"; fi
```

Comment:
should not version be assigned before being used in pkg name in if block?

Also on else don't we need _ before version?